shrine_with_cart/mdc_100_series/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ dependencies:
   intl: ^0.15.6
 
   cupertino_icons: ^0.1.0
-  scoped_model: 0.2.0
+  scoped_model: 0.3.0
```

Comment:
Technically should be in another PR. NBD

Reply:
Ok!